app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -702,12 +702,13 @@ function AcuantCapture(
           allowUpload &&
           formatHTML(t('doc_auth.buttons.take_or_upload_picture_html'), {
             'lg-take-photo': () => null,
+            'lg-or': ({ children }) => (
```

Comment:
I struggled with changing this component recently. If you do come up with a way to pull it apart, I'd support that being part of this PR.

Reply:
@charleyf , personally I found it is very handy.

Reply:
Oh, interesting. Can you say more about that?